daemon/exec.go

```diff
@@ -306,6 +307,25 @@ func (daemon *Daemon) ContainerExecStart(ctx context.Context, name string, stdin
 	return nil
 }
 
+// ContainerExecKill kills an exec instance.
+func (daemon *Daemon) ContainerExecKill(ctx context.Context, name string, sig uint64) error {
+	e, err := daemon.getExecConfig(name)
+	if err != nil {
+		return err
+	}
+
+	if sig != 0 && !signal.ValidSignalForPlatform(syscall.Signal(sig)) {
+		return fmt.Errorf("The %s daemon does not support signal %d", runtime.GOOS, sig)
```

Comment:
This error still needs to be wrapped in an `errdefs.InvalidParameter`
Also error messages should start lower-case.